tools/run_tests/performance/scenario_config.py

```diff
@@ -140,13 +146,23 @@ def _ping_pong_scenario(name, rpc_type,
     if unconstrained_client == 'async':
       deep = DEEP
       wide = WIDE
+      num_clients = 0 # use as many clients as available
     elif unconstrained_client == 'sync':
       deep = SYNC_DEEP
       wide = SYNC_WIDE
+      num_clients = 0 # use as many clients as available
+    elif unconstrained_client == '1chan_bw':
```

Comment:
nit: does bw stand for bandwidth?  I think just 1channel_bandwidth would be more readable here.
Just a nit, so fine to merge for now.